Form-Controls/index.html

```diff
@@ -12,16 +12,75 @@
       <h1>Product Pick</h1>
     </header>
     <main>
+      <style>
+        body {
+          font-family: Arial, sans-serif;
+          line-height: 1.6;
+          max-width: 600px;
+          margin: 0 auto;
+        }
+        form {
+          display: flex;
+          flex-direction: column;
+          gap: 1.5rem;
+        }
+        label {
+          font-weight: bold;
+        }
+      </style>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <form action="/submit" method="post" novalidate>
```

Comment:
I don't see `/submit` in your project. Please use `actin="#"` which is a convention to say "as for now we don't care where this form is submitted"